VirtoCommerce.CatalogModule.Web/Scripts/blades/property-list.js

```diff
@@ -27,7 +27,7 @@ angular.module('virtoCommerce.catalogModule')
             $scope.resetFilter();
 
             if ($localStorage.entryPropertyFilters) {
-                var savedFilteredProperties = $localStorage.entryPropertyFilters[authService.id];
+                var savedFilteredProperties = $localStorage.entryPropertyFilters[authService.id][blade.currentEntity.categoryId];
```

Comment:
Thus we are changing the requirement that we are saving the filter for the user for any catalog.
I think we need to discuss it with Nikolay, as it was discussed before and proposed to save by categories, but we decided to choose one filter for the user.

Reply:
Fixed